.github/workflows/main.yml

```diff
@@ -0,0 +1,26 @@
+on: [push, pull_request]
+name: Test
+jobs:
+  test:
+    strategy:
+      fail-fast: false
```

Comment:
Why `false`? Sounds like failing fast shortens the wait.

Reply:
This is fail-fast for jobs, not workflows. Failing fast stops any in-progress parallel jobs. Since we want to parallelize running tests on Linux and Windows, it's [recommended](https://ncorti.com/blog/howto-github-actions-build-matrix) to turn that off.